### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by hislordshipprof**
-  Add a new JavaScript file and import it as a module:
-  Status updates (completed: true / false).
-  Added event listener to the checkbox (change).
-  Update items object's value for completed key upon user actions.
-  Implemented a function for the "Clear all completed"

### CODE REVIEW

File: src/crudOps.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  // get todo items from local storage
+  static getTodos = () => JSON.parse(localStorage.getItem('todos')) || [];
+
+  // add todo item to local storage
+  static addTodo = (todo) => {
+    const todos = this.getTodos();
+    todos.push(todo);
+
+    localStorage.setItem('todos', JSON.stringify(todos));
+  };
```
Comment (DeliceLydia) [inline src/crudOps.js:11]: - [ ] I am not able to add a `to do` that means I am not able to test other methods please address that

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @hislordshipprof, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights for your project:


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/crudOps.js (hunk 1 above)
Comment (DeliceLydia) [reply src/crudOps.js:11]: - [ ] Remember to update your README

Comment (hislordshipprof) [reply src/crudOps.js:11]: please get the project started using npm start on ther server

Comment (hislordshipprof) [reply src/crudOps.js:11]: readme updated

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @hislordshipprof , 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️

- [x] As the previous reviewer earlier noted, i could not add a todo-item. I noticed that the form was not submitted, hence why the value is not visible in local storage. Please check it out.

_Check the comments under the review._

## Optional suggestions
- Kindly import your javascript file as a module into your HTML file as specified in the requirement

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by hislordshipprof** [comment]
Please i stated that to get the project running you need to do npm start using the liveserver wont work. Kindly note that we are using webpack and live server wont work with it.
kindly use npm start to get it running and you would be able to add it.. its working here

**Review by vickymarz — APPROVED** [review]
Hi @hislordshipprof , 

# 🟢  Status: APPROVED 🏆 🎉 🎊 

Thanks for your understanding and patience. You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**